lib/ts/PriorityQueue.h

```diff
@@ -110,6 +110,12 @@ PriorityQueue<T, Comp>::pop()
     return;
   }
 
+  // SKH - I suspect this assignment is not preserving entry indices correctly.
```

Comment:
Actually, I think that pop() is almost always safe since the last entry is largest and the swaps in bubble_down will fix up the indices.